src/actions/loginActions.js

```diff
@@ -4,6 +4,7 @@
 import toastr from "toastr";
 
 export const loginUrl = "https://ah-backend-thor.herokuapp.com/api/users/login/";
+// export const loginUrl = "http://localhost:8000/api/users/login/";
```

Comment:
May you kindly remove the localhost url comment.

Reply:
still testing, will update it after the test